packages/commonwealth/client/scripts/models/Thread.ts

```diff
@@ -462,7 +462,7 @@ export class Thread implements IUniqueId {
           parent_id: null,
           reactions: [],
           CommentVersionHistories: [],
-          reaction_weights_sum: '0',
```

Comment:
i think this was intentional in https://github.com/hicommonwealth/commonwealth/commit/7e56b0c11d4cc41e3b746d19940985ea250b26c1#diff-65f6a84029354b4ea2d8a68ac1b9453ee76725ba26af0e19af0e1f582b9083d2

cc: @rbennettcw to confirm.

Reply:
oops yea will revert that